### PR TITLE
re: Update index.html

### PR DESCRIPTION
**Pull request by itsN1X**
Request for Comments

@asdofindia @debloper @kaustavdm @iamvp7 @iamjayakumars @soulspark666

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,6 +214,14 @@ <h1 class="title">
                                         <a href="https://github.com/MozillaTN">GitHub</a>
                                         <a href="https://twitter.com/mozillaTN">Twitter</a>
                                     </div>
+                                    <div class="box">
+                                        <p class="subtitle">vMozilla</p>
```
Comment (obitodarky) [inline index.html]: I think it's better to name regional groups after cities/states so that others can recognize what region it is

File: index.html [hunk 2]
```diff
@@ -215,6 +214,14 @@ <h1 class="title">
...
+                                        <a href="https://t.me/joinchat/IQStrQxhC8M5S0TD3VQkOg">Telegram</a>
+                                        <a href="https://twitter.com/vMozilla">Twitter</a>
+                                        Facebook<a href="https://www.facebook.com/MozillaVIT">Page</a>
```
Comment (obitodarky) [inline index.html]: This looks a bit off. One suggestion is you can add two links: "Facebook Page" and "Facebook Group"

Screenshot 2019-05-28 at 9 16 25 AM

File: index.html (hunk 1 above)
Comment (soulspark666) [reply index.html]: I think so he has added Mozilla community for Vellore Institute of Technology, Vellore, so I don't think that it can be named as a regional group.

Comment (obitodarky) [reply index.html]: From Telegram discussions, it seems that Mozilla Community for VIT itself is the regional group(pardon me if I misunderstood) which again raises the question that should it come under regional groups, if so then should it be named after the place rather than the group?

File: index.html (hunk 2 above)
Comment (itsN1X) [reply index.html]: image

How about superscripting? `the one in right col.`
Or Facebook[Page]()|[Group]()

Comment (obitodarky) [reply index.html]: Superscripting seems fine. I think it gets the work done

File: index.html (hunk 1 above)
Comment (itsN1X) [reply index.html]: Yep, after discussing over telegram, I've realized its in everyone's best interest to call the umbrella Community as `vMozilla` :1st_place_medal: 

The Campus Clubs are therefore shortened to `vMozilla <Geolocation>` :2nd_place_medal: 

-----

So, I'll have a placeholder commented out for future (upcoming) CCs at other @VITUni campuses.

## TL;DR : All communities referencing @VITUni <
# New names are shorter, meaningful and #include 'Mozilla' too!
::|Name|Purpose|Notes
----|----|----|----
RC-1|vMozilla|Umbrella Community|Incl. alumni/staff/enrolled/associates to @VITUni)
CC-1|vMozilla, Vellore|Campus Club|Incl. enrolled volunteers to @VITUni-#Vellore)
CC-2|vMozilla, Chennai|Campus Club|Incl. enrolled volunteers to @VITUni-#Chennai)
CC-3|vMozilla, Amrawati|Campus Club|Incl. enrolled volunteers to @VITUni-#Amrawati)
CC-4|vMozilla, Bhopal|Campus Club|Incl. enrolled volunteers to @VITUni-#Bhopal)
CC-5|vMozilla, Jaipur|Campus Club|Incl. enrolled volunteers to @VITUni-#Jaipur)

**Comment by itsN1X** [comment]
image
Prospect↑.
Reviews requested.

File: index.html (hunk 1 above)
Comment (itsN1X) [reply index.html]: Resolves : :+1: 

- [x] Naming of Campus Club (@VITUni-<GeøŁøcation>)

File: index.html (hunk 2 above)
Comment (itsN1X) [reply index.html]: Resolved : :+1: 

- [x]  ::  SuperScript’d

**Comment by asdofindia** [comment]
I took a look at the changes. Some suggestions:

- Change "vMozilla" to "vMozilla (Mozilla `region`)" where region is the region it pertains to. If it doesn't pertain to any particular region, I think it doesn't qualify for the regional groups section. Maybe it can then be in the functional groups?
- Superscripting does not seem like the right way to do similar links. Perhaps we can have all links on newlines? I apologize for not thinking about this earlier. Check this out:
image
Maybe a designer can suggest a better way
- Why does vMozilla-Vellore have 3 facebook pages and 2 websites? Perhaps you can include the official/updated page and website alone? Remember that one campus cannot have more than one campus clubs as per Mozilla requirements. If both blogspot and wordpress are being maintained, perhaps there can be links like "Wordpress" "Blogspot" instead of Website 1, 2

**Comment by obitodarky** [comment]
- I agree, I also don't think it qualifies as a regional group since it covers Vellore as well as Chennai. I think it a better way to put it is that `vMozilla` is a group that comes under `Mozilla TN` regional group (which I believe does exist).Functional groups could be the way to go.

- I think newlines would increase the length of the page unneccessarily. I believe @pransh15 should have some idea as for how the sections should go.

**Comment by itsN1X** [comment]
> I took a look at the changes. Some suggestions:
> 
>     * Change "vMozilla" to "vMozilla (Mozilla `region`)" where region is the region it pertains to. If it doesn't pertain to any particular region, I think it doesn't qualify for the regional groups section. Maybe it can then be in the functional groups?
I believe Campus Clubs is an initiative and not the community in itself.
We currently have 50 members who contribute to mozilla activities, but are, essentially, unaware of each other.
Having an umbrella body, `vMozilla` would only consolidate volunteer focii on right locii.

I think '**vMozilla, India**' is, _then_, more apt - a noun when a functional group(?).
> 
>     * Superscripting does not seem like the right way to do similar links. Perhaps we can have all links on newlines? I apologize for not thinking about this earlier. Check this out:
>       image
>       Maybe a designer can suggest a better way
Maybe. It could pass, or then we could go overboard and put svg/icons, with tooltips.
> 
>     * Why does vMozilla-Vellore have 3 facebook pages and 2 websites? Perhaps you can include the official/updated page and website alone? Remember that one campus cannot have more than one campus clubs as per Mozilla requirements. If both blogspot and wordpress are being maintained, perhaps there can be links like "Wordpress" "Blogspot" instead of Website 1, 2
So, basically : Mozilla is open-source and nobody is obligated to anything IRL.

It (the myriad sites, pages, articles) could all be properly archived if organized / stashed into a single place/bucket so that there is a timeline (smh).. Contributors will aIways come and go; to make the collab count, it must be re:viewed. But what has happened (from lay observations) is that every quarter/2-4 months, new groups assimilate themselves into event-conductors, et. al., and after the 'event', there is no trail!

## TL;DR
I think CCs are grassroots initiatives.. and more keen on the marketing frontier..
The RCs, imho, could be anything, from big to small, or even if we allow, single-person thing.. and there job should be nurturing the CCs..

Another great factor to include in this discourse is the age-group of our participants.
Few people over 25 attend university.
Few people under 25 understand RUST.
In my eyes, maximizing of contribution/participation will commence only when people actually make opensource open for people instead of just their own sake. I mean actually have a dialogue instead of presuming your purposes.
What I assert here is new contributors are more prone to wear their enthusiasm off than seasoned contributors.

Not sure if any of this will make sense, but yep, I'd love to learn what happens if it does.

###### This issue will be left open ti̶ll̶
~ 10 days to discuss might yeild.

**Comment by obitodarky** [comment]
I don't think this is the right place to discuss the `Campus Clubs and their guidelines` topic in detail. Leaving this topic on [discourse](https://discourse.mozilla.org/) should be a better idea. As for the pull request, if there are any potential changes and reviews then we can keep this PR open otherwise, I think this issue should be closed.

**Comment by asdofindia** [comment]
@itsN1X I think https://discourse.mozilla.org/t/reinventing-mozilla-on-campus/8730/ and https://discourse.mozilla.org/t/frequently-asked-questions-faq/10691 are very relevant links. The latter actually says

> "Part of the definition of a club is that they meet regularly in person, while some activities will definitely be virtual, in order to be recognized as a Mozilla Club they need to be primarily based in the real world." 

You are suggesting an "umbrella" group for Campus Clubs. It goes against the concept of campus club. If people are interested in meeting other contributors of Mozilla they should join official Mozilla groups. Infinitely looping on creation of sub-groups and umbrella groups does not make sense (at least to me).

Re: "nobody is obligated to anything IRL". Unfortunately I don't quite agree with this. Firefox is open source. Mozilla is a registered trademark of Mozilla. If you're using the trademark, you should be in compliance with whatever guidelines Mozilla lays down. Think of it. What if there is an evil element who starts a "Mozilla" group, say "xxxMozilla" and proceed to promote things antithetical to the idea of open web? Who gives them the authority to use Mozilla's name and goodwill associated with it to their own benefit? Mozilla has laid down clear guidelines on campus clubs and this will have to be followed.

Now, Mozilla doesn't quite have a mechanism to officially register regional groups. There was/is the [events registration pilot](https://events.mozilla.org/pilot) which could end up as the official mechanism of registering groups. It lays down certain criteria

> Are Established
> 
> For the pilot we want to learn how we can support the groups who are already around. We love that you want to start a new group but we'll ask you to hold off until after the pilot to apply.
> 
> Hold Regular Meetings
> 
> They can be in-person or virtual, but we want to support groups that have a regular cadence of getting together to coordinate, plan, and work.
> 
> Contribute to Mozilla
> 
> Obviously! You need to make regular contributions to projects that support Mozilla's mission, whether it be localizing, answering support questions, throwing events, or participating in campaigns. There must be a contribution every 6 months to remain 'active'
> 
> Have at Least 3 Members
> 
> Hanging out with a few friends is great but in order to be registered community groups must have an active set of contributors 3+ members.
> 
> Are Open and Welcoming
> 
> Mozilla and it's official groups welcome everyone who shares Mozilla's goals and wants to contribute in a healthy and constructive manner to participate. Groups must be easy to join and cannot restrict membership.
> 
> Follow the CPG
> 
> The heart of Mozilla is people as such we require all groups who participated to agree to adhere and uphold to the Community Participation Guidelines (CPG).

But this has not quite been put in place officially. It is still a pilot. Then, where do these "regional groups" and "functional groups" come from? Well, you will have to turn back to the concepts that was agreed upon in the last [Mozilla India meetup](https://leanpub.com/mozillaindia-openbook/read) for that. But is this relevant anymore considering that most of it has not been implemented? I don't know.

You could still go ahead and form your own "Mozilla" communities. This is exactly what has been happening forever. Should your own community find a spot in mozillaindia.org website, though? Well, if that has to happen, there should be some kind of an agreed upon process to it.

Your thought on adding vMozilla as a functional group makes sense in the above context. The function here could be coordination/communication between campus clubs. Perhaps you can add a description below the name as "group for campus club members to interact with each other" or something that describes and defines the group.

**Comment by asdofindia** [comment]
> It (the myriad sites, pages, articles) could all be properly archived if organized / stashed into a single place/bucket so that there is a timeline (smh).. Contributors will aIways come and go; to make the collab count, it must be re:viewed. But what has happened (from lay observations) is that every quarter/2-4 months, new groups assimilate themselves into event-conductors, et. al., and after the 'event', there is no trail!

Imagine this from the perspective of a new contributor. Which is *the* website they should go to? Which group should they post things in? The right spot for archive websites of a club would be on the current website of the club (and not on mozillaindia.org listing). If there are two websites being maintained in parallel, no issue in adding them both with a separate descriptive name. For example, if one of the website acts as official website and the other acts as a blog, name them as "Website" and "Blog" respectively. I hope you understand that I'm not trying to obstruct contributions, rather facilitating long term health.

**Comment by soulspark666** [comment]
I agree with @asdofindia

> Which is the website they should go to? Which group should they post things in? The right spot fo...........

As it will only cause confusion and will eventually lead people confused at the time of help.

As a learning web developer, I think super scripting is a good solution for the time being.

**Comment by itsN1X** [comment]
>> @adsofindia:

> Your thought on adding vMozilla as a functional group makes sense in the above context. The function here could be coordination/communication between campus clubs. Perhaps you can add a description below the name as "group for campus club members to interact with each other" or something that describes and defines the group.

Thank you for making me sound **more** clear.
> The function here could be coordination/communication between campus clubs.

Yep. Their short-lived (mostly a fortnight) efforts need to be put in a time-bottle.
Essentially, this also helps delimit unused-software/code authoring. Even provides pointers to fresh people looking for inspiration.

###### Its still 1559B s, I want to wait till 1560B s (8<sup>th</sup>ju) and know of more opinions.

----
I request this PR be left open for concerned mozillians to discourse.

* Upmerge (the VU additions) will not be done before friday-midnight (7th).
* `vMozilla`, being, a **Functional** Group, &not _Regional_ Group, is thus, looking `final`.

----
#### Feel free to leave a suggestion or comment out your viewpoint on he added code-comment in the PR.
Useful links: <details><summary>The Mozilla India Open Book</summary>

Table of Contents

        Introduction
            About
            TL;DR
        Organization structure
            Mozilla India Restructure Proposal
            FAQ

Version   : 0.3.0

Published : 2016-09-12

Introduction
About

This book serves as the canonical document for the organizational structure adopted by the Mozilla India community.

It is collaboratively written by Mozillians. The Meta Team of Mozilla India serve as its editors and maintainers.
A bit of history

This book was put into place right after the Mozilla India Meetup at Pune in August, 2016, to help document the outcome of the efforts to restructure the Mozilla India.
Contribute

The source code of this book is hosted publicly on Github. You are welcome to put forward your suggestions, edits or additions to the book. If you are interested, take a look at the repository.
TL;DR

Here is a quick summary of the currently proposed organizational structure for the Mozilla India community. If you are interested in the full proposal, skip to the next chapter.
Proposal summary

Have open, transparent, accountable teams where individual contributors can work together with the highest amount of flexibility. Adopt a participatory process that does not add excess process overhead. Everyone is urged to take up the things they like and work together with others, and in the process, improve the structure itself.

Overall team composition consists of “Functional Teams” (with long term focus) and “Focus Teams” (possibly cross-functional, with short term focus on particular events/use cases). These teams are formed around functional contribution areas, with a degree of alignment with Mozilla’s key concerns.

The “Meta Team” glues things together and ensures smooth function and cross-team collaboration. The teams are supported by various infrastructure and communication channels.

Each team will have two “facilitators” who will act as the communication interface of the team with the outside world. They will take up accountability on behalf of their teams.

There are also “Geographic Groups” that consist of Mozillians in particular regions and help in discovering and connecting Mozillians by location.

Ownership is on everyone for the roles they take up. Leadership builds as a result of achieving expertise through constant practice and grows organically.

Self-evaluation, backed by data, is done at every point to understand effectiveness and impact of the community and its initiatives.

This is a constantly evolving, fluid and flexible structure. Nothing is set in stone.
Organization structure
Mozilla India Restructure Proposal
Vision for the community

Enable the community to uphold the Mozilla mission at any scale, while remaining relevant to individual contributors.
Fundamental principles to help achieve the vision

    Maintain transparency of operations (ensures openness)
    Invite participation explicitly (includes trust-first model)
    Decentralize decision making
    Maintain ubiquity of communications
    Ensure accountability through clear metrics and documentation of actions and decisions.
    Use flexible and adaptive structure which keeps on improving itself through constant feedback and evaluation.

Code of Conduct

The community code of conduct will adopt the very same Mozilla’s up-to-date CoC.
Organizational Structure
Basics

    Any non-general role will be shared by 2 persons to ensure redundancy of human resources and contingency plan.
    Each team in the following structure tree will have a role of facilitator (shared by 2 people)
        They work as the communication interface for the team with the outside world and the meta team
        They work as the arbitrators to help finalize team decisions (and save from it needing to bubble up to meta team)
        They ensure execution of the undertaken responsibilities of the team and publish activity reports publicly and to the overseeing body
        In any case either of the facilitators discontinues or becomes unavailable for the role, a second facilitator is chosen from the team within a preset timeout
        Any concern about the team’s health, deliverable, execution should be resolved by the facilitators, and to be flagged to the overseeing team if failed within preset timeout
        Any concern about peer teams’ health, operation, and execution should also be noted, reviewed and resolved OR flagged to the common overseeing team, as soon as observed
    Each team can have their own preferred internal communication channels, but they would have to conform to the community preferred channels to document, broadcast and publish activity reports
    Teams need to be formed with clear objectives.
    While choosing team members, teams should look out for the best suited, capable and passionate individuals who are willing to pursue the objectives of the team.
    Community’s operation process will follow a meritocratic processes instead of a democratic process.
    Every part of the community should have well documented and discoverable waypoints/pathways
    Self-checks at any level and at any given time is encouraged, and decided resolution is applied to evolve the community structure on the fly.
    Every entity (individual or team) in the community starts with a preset level of trust
    The level of trust allows for different level of responsibilities to be undertaken
    The trust level can be gained or lost, based on the activities/track-records

Hierarchy
Meta Team

    Core of the community identity
    Experience based membership
    Sets goals, allocates resources
    Oversees community operations
    Resolves conflicts and grievances
    Is accountable to the community through a self-evaluation report at least once every 6 months.

Functional Teams

    Persistent teams
    Strategic objectives
    Limited memberships
    Responsible for functional growth
    Overseen by Meta Team, peer reviewed by other functional teams

Focus Teams

    Temporary teams
    Tactical objectives
    Scalable Memberships
    Responsible for campaign deliverables
    Overseen by participating functional teams

Geographic Groups

    Persistent group
    Core purpose is to enable functions that benefit from geographic coordination
    Membership is by virtue of someone’s presence in that geographic area, and it will be seamless for any person to move to a new geographical group
    Supported by Meta Team

Communications and PR

A set of centralized communication media will be necessary for the community’s internal discussions and external outreach. The goal of such a communication system is to encourage visibility of information while remaining relevant for focused group discussions.

    Discourse for discussions
        Threaded, searchable, near-real-time
        Can be piped to our existing community-india mailing list.
        Sub-categories can help in grouping discussions
    Telegram for real-time discussions
        Real-time discussions that do not need to be threaded.
    Wiki for canonical resources
        collaboratively editable, references, non-real-time
        Can use wiki.mozilla.org/India
    Blog.MozillaIndia
        PR, outreach, announcements
    Social media
        PR and engagement - outreach, announcements

Infrastructure accesses required

    DNS & Email Liaisons
    Web server access
    Discourse admins
    Mailing list admins
    Telegram admins
    Blog admins
    Wiki admins
    Social network admins
    Facebook groups and pages
    Twitter access
    YouTube admins
    Instagram access
    Bugzilla admins
    Github organization access groups

Recognition

    Individual activity metrics, funneled from various resources will be listed publicly and will serve as the primary medium of recognition.
    This process will require gradual improvement to be more concise as more and more activity facets are discovered and added in.
    Individual rewards will be for Mozilla to distribute; the local community will only ensure availability of the data for decision making.

Self-checks, grievances and arbitration

    Self-checks can happen in any level, by anyone, by using data and activity log with references.
    For any issue i.e. coming from the self-checks should be raised to the concerned bodies.
    Failing to resolve any issue flagged at any level should be documented and escalated.

*    *    *

    There will be different ways to put forward a grievance for resolution, by different order of anonymity.
    Grievances (breaking of CoC, inter-personal discomfort, reporting peers/processes etc) has to be directly escalated to the meta team.
    The lack of escalation steps is there to ensure homogeneous treatments for the issues raised, and not bottleneck at any steps or go unresolved.

*    *    *

    Arbitration process for any conflict should follow a reason and argument based process.
    Arbitration process should only come into effect if the members of a team fail to resolve a conflict.
    Arbitration process should ensure the vision and fundamental principles in order to achieve appropriate resolution.
    At any step/team if the arbitration process fails to resolve, it is then immediately escalated to the overseeing team to take up.

>**FAQs**
What do you mean by transparency?

Talk and work in the open
>What do you mean by being explicit about invitation?

Explicitly welcome contributions by documenting things which will make onboarding simpler. Announce on channels where people are listening.
>What does decentralized decision making mean in this context?

No central person for making any decisions related to the team or the larger community.
>What does ubiquity mean here?

Present everywhere, on the channels that are decided to be the primary communication channels of the entire community.
>What are self-checks?

Self-evaluation, which includes introspection (analyze your own ideas and thoughts) and retrospection (analyze the past and identify things which can be improved). This can be done both for yourself or for a team or, even, for the larger community.
>When can we start using the new structure?

It would take at least a couple of months to get the implementation in order, but it is a self-checking model in which we would keep on making the amendments to make it better.
>What happens to the task-force model?

The current task force teams have been dissolved.
>Who sets goals for functional teams?

Functional teams have their own goals which is overviewed by the Meta Team to keep them aligned to the global Mozilla standards.
>How would a new person be selected in a team?

Each team will have their own selection criteria depending on the nature of contribution required. Any new team member would be selected depending on the team decision and the selection criteria would be documented in future. For focus teams, there is no limit for the number of members.
>Can one person join multiple functional teams?

There is no rule against one person joining multiple functional teams, but they are encouraged to join a single team due to amount of dedication required. However, they are free to contribute to other teams as well.
</details>
<details><summary>https://discourse.mozilla.org/t/frequently-asked-questions-faq/10691</summary>

>I heard that Clubs are shutting down. Are Mozilla Campus Clubs also shutting down and what is the difference between Mozilla Clubs and Mozilla Campus Clubs?

No, Mozilla Campus Clubs program is not shutting down, it will continue to be supported and staffed by the Mozilla Corporation. Local groups with a passion for technology who want to work together to build the open web on University and College campuses are invited to register and participate through the campus clubs program at <https://campus.mozilla.community>.

* Campus Clubs exist on University campuses with a focus on technology and Open Source and engage only students from the campus.
* Whereas, the Mozilla Clubs existed outside campuses with a focus on reading, writing, and participating on the web.
</details>

**Comment by asdofindia** [comment]
Please remove commented code before it can be merged.

**Comment by obitodarky** [comment]
> Please remove commented code before it can be merged.

@itsN1X  is keeping the PR open till 7th of June for more suggestions on this.

**Comment by itsN1X** [comment]
> Update local::branch to reflect.
#include #=#130

**Comment by asdofindia** [comment]
@itsN1X, your comments are too cryptic for me. If you're trying to convert this pull request into a form that can be merged, I request you to do these:
- [ ] remove commented/dead code.
- [ ]  squash commits into one (optional, as I can do this while merging)

**Comment by obitodarky** [comment]
@itsN1X please make the necessary changes in order to merge the pull request

**Comment by asdofindia** [comment]
@itsN1X https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History has some helpful information on rewriting git history.

**Comment by itsN1X** [comment]
> One of the great things about Git is that it allows you to make decisions at the last possible moment.

⇨  🔛

###### I have been putting it off and yet made my patch concurrent with all upstream changes to the `main::master`, and will soon merge-in, verbatim, if no follow-up emerges from within the VIT Community.
##### Will advent further to extract more responses regarding future administration. Guess this needs time, quality discussions.

**Comment by itsN1X** [comment]
> @itsN1X https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History has some helpful information on rewriting git history.

Thank you for sharing a useful tip!
###### Will have it shared furthermore.

**Comment by itsN1X** [comment]
> Can you please share a screenshot?

[gh-pages](https://github.com/itsN1X/mozillaindia.github.io/commit/ddf1b7df68b3f3f80f6b8a295c094d17d9e34523#comments)🍥

**Comment by obitodarky** [comment]
Closing this since it is no longer relevant to the new gatsby branch.

**Comment by itsN1X** [comment]
_roger that_
